tsl/src/remote/dist_ddl.c

```diff
@@ -980,10 +980,6 @@ dist_ddl_process(const ProcessUtilityArgs *args)
 			dist_ddl_process_truncate(args);
 			break;
 
-		case T_CopyStmt:
-			/* Nothing to do for COPY. */
-			break;
-
```

Comment:
Why does adding a new GUC need this change?

Reply:
That is a leftover from previous work, this case is never reached because it is checked at the beginning of the ddl start function